metricbeat/index.json

```diff
@@ -0,0 +1,10962 @@
+{
+  "settings": {
+    "index.number_of_shards": {{number_of_shards | default(1)}},
+    "index.number_of_replicas": {{number_of_replicas | default(0)}},
+    "index.mapping.total_fields.limit": 20000,
+    "index.search.slowlog.level" : "debug",
+    "index.search.slowlog.threshold.fetch.debug" : "0s",
+    "index.search.slowlog.threshold.query.debug" : "0s"
+  },
+  "mappings": {
```

Comment:
Meanwhile types have been removed on master so this will issue deprecation warnings. I have extracted the Metricbeat mapping from 7.0.0-beta1 in https://gist.github.com/danielmitterdorfer/72686146100dea7b775f0efdc80bd2f1. Can you please use that mapping instead so we avoid the deprecation warnings?